### ISSUE TITLE
Hello, excuse me, could you please provide us with your test.py?

### ISSUE THREAD

**Issue by HIT-wwb** [junbopeng/CBCT2CT_cDDPM#3]
A code that evaluates the performance of the algorithm using the Mean Absolute Error (MAE), Peak Signal-to-Noise Ratio (PSNR) and Normalized Cross-Correlation (NCC) metrics of synthetic CT (sCT) samples.Thank you very much

**Comment by qiangzhang649** [comment]
I want too. Thank you very much

**Comment by HIT-wwb** [comment]
@qiangzhang649 Hello, is there any progress on your test code?

**Comment by qiangzhang649** [comment]
Yes, why is the background of the images generated during inference black? I encountered the same problem as others. Could you provide me with the inference code? Thank you!

**Comment by junbopeng** [comment]
> Yes, why is the background of the images generated during inference black? I encountered the same problem as others. Could you provide me with the inference code? Thank you!

The script of test_conditon.py has been uploaded.
